.buildkite/pipeline.yml

```diff
@@ -12,5 +12,12 @@ steps:
   - label: Build the pex file
     command: mkdir -p dist && .buildkite/build_pex.sh
 
+  - wait
```

Comment:
Do we need a wait on this? It would be nice if installer building can run in parallel as much as possible.